server/game/gamesteps/challenge/fulfillmilitaryclaim.js

```diff
@@ -10,15 +10,24 @@ class FulfillMilitaryClaim extends BaseStep {
     }
 
     continue() {
-        var promptMessage = 'Select ' + this.claim + ' ' + (this.claim > 1 ? 'characters' : 'character') + ' to fulfill military claim';
+        this.forcedClaim = _.filter(this.player.mustChooseAsClaim, card => card.controller === this.player && card.location === 'play area');
+        
+        let claimToSelect = this.claim;
+        
+        if(this.forcedClaim.length < this.claim) {
+            claimToSelect = this.claim - this.forcedClaim.length;
+        }
+
+        var promptMessage = 'Select ' + claimToSelect + ' ' + (claimToSelect > 1 ? 'characters' : 'character') + ' to fulfill military claim';
```

Comment:
Since this variable is being moved and others are being declared in this method, `let` instead of `var`.